ts/IterableInboxMessageDisplay.tsx

```diff
@@ -31,6 +32,25 @@ const IterableInboxMessageDisplay = ({
    const messageTitle = rowViewModel.inAppMessage.inboxMetadata?.title
    const [inAppContent, setInAppContent] = useState<IterableHtmlInAppContent>(new IterableHtmlInAppContent(new IterableEdgeInsets(0, 0, 0, 0), ""))
 
+   let sampleHTML = `
```

Comment:
This can be removed now right?

Reply:
Yes, for this PR, I will remove all functionality associated with the in-app HTML links.